src/main/java/hudson/plugins/disk_usage/DiskUsageCalculation.java

```diff
@@ -56,20 +54,14 @@ public String getThreadName() {
     public abstract DiskUsageCalculation getLastTask();
 
     public long scheduledLastInstanceExecutionTime() {
-        try {
-            if(getLastTask() == null || getLastTask().isCancelled()) { // not scheduled
-                return 0L;
-            }
-            long time = getCronTab().ceil(new GregorianCalendar().getTimeInMillis()).getTimeInMillis();
-            if(time < new GregorianCalendar().getTimeInMillis()) {
-                return 0;
-            }
-            return time;
-
-        } catch (ANTLRException ex) {
```

Comment:
To preserve existing behavior, should catch `IllegalArgumentException` instead of `ANTLRException`.